File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/menutext/vb/guids.vb
Imports System

Class GuidList
    Private Sub New()
    End Sub

    Public Const guidMenuTextPkgString As String = "092eefe9-d7de-4281-9965-db64580fb157"
    Public Const guidMenuTextCmdSetString As String = "c369d294-d41f-450e-bef7-7571532014a1"

    Public Shared ReadOnly guidMenuTextCmdSet As New Guid(guidMenuTextCmdSetString)
End Class

File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/menutext/cs/guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace Microsoft.MenuText
{
    static class GuidList
    {
        public const string guidMenuTextPkgString = "b1253bc6-d266-402b-89e7-5e3d3b22c746";
        public const string guidMenuTextCmdSetString = "a633d4e4-6c65-4436-a138-1abeba7c9a69";

        public static readonly Guid guidMenuTextCmdSet = new Guid(guidMenuTextCmdSetString);
    };
}